twilight-model/src/guild/onboarding/prompt_type.rs

```diff
@@ -0,0 +1,66 @@
+use serde::{Deserialize, Serialize};
+
+/// The type of an onboarding prompt.
+#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(from = "u8", into = "u8")]
+pub enum OnboardingPromptType {
+    /// A prompt that allows the user to select multiple options.
+    MultipleChoice,
+    /// A prompt that allows the user to use a dropdown to select an option.
+    DropDown,
```

Comment:
[Discord Docs](https://discord.com/developers/docs/resources/guild#guild-onboarding-object-prompt-types) documents this as one-word, so it should be renamed to `Dropdown`